### DISCUSSION TITLE
How to add external url

### DISCUSSION THREAD

**Discussion by maurokenny** [prefix-dev/pixi#1197] [General]
Hi,

How can I replace the following lines, but using pixi, please:

`conda create --solver=libmamba -n rapids-23.12 -c rapidsai -c conda-forge -c nvidia rapids=23.12 python=3.10 cuda-version=12.0`
`pip3 install torch==2.1.1+cu121 torchvision==0.16.1+cu121 torchaudio --extra-index-url https://download.pytorch.org/whl/cu121`

**Comment by vigneshmanick** [comment]
`extra-index-url` is currently not supported, see https://github.com/prefix-dev/pixi/issues/1094
